### PR TITLE
[Improvement-#11768][Monitor] Support monitor h2 database in monitor page

### LINKED ISSUE #11768: [Improvement][Monitor] Support monitor h2 database in monitor page
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Right now, we only support monitor mysql/pgsql in monitor page, when we start at standalone mode, we cannot get the db health in monitor page, it's better to support h2.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by DarkAssassinator**
…1768

## Purpose of the pull request

fix #11768 

## Brief change log

add H2 db performace monitor 

## Verify this pull request

This pull request is code cleanup without any test coverage.

Just add connection sessions monitor, becasue H2 DB do not have MAX_CONNECTION setting.

### CODE REVIEW

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        MonitorRecord monitorRecord = new MonitorRecord();
+        monitorRecord.setDate(new Date());
+        monitorRecord.setDbType(DbType.H2);
+        monitorRecord.setState(Flag.YES);
+
+        Statement pstmt = null;
+
+        try {
+            pstmt = conn.createStatement();
+            try (ResultSet rs1 = pstmt.executeQuery("select count(*) as total from information_schema.sessions;")) {
```
Comment (SbloodyS) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java]:
```suggestion
            try (ResultSet rs1 = pstmt.executeQuery("select count(1) as total from information_schema.sessions;")) {
```

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    @Override
+    public MonitorRecord getMonitorRecord(Connection conn) {
+        MonitorRecord monitorRecord = new MonitorRecord();
+        monitorRecord.setDate(new Date());
+        monitorRecord.setDbType(DbType.H2);
+        monitorRecord.setState(Flag.YES);
+
+        Statement pstmt = null;
+
+        try {
```
Comment (ruanwenjun) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java]: Please use try-with-resource to close the `Statement`

Comment (DarkAssassinator) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java]: > Please use try-with-resource to close the `Statement`

done. thx.

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java (hunk 1 above)
Comment (DarkAssassinator) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/H2Performance.java]: > 

done. but i still think that count(1) ≈ count(*)

**Review by caishunfeng — APPROVED** [review]
LGTM

**Review by ruanwenjun — APPROVED** [review]
+1

**Pull request merged**